Foxtrot/Driver/Rewriting/EmitAsyncClosure.cs

```diff
@@ -59,6 +58,121 @@ namespace Microsoft.Contracts.Foxtrot
     /// </remarks>
     internal class EmitAsyncClosure : StandardVisitor
     {
+        /// <summary>
+        /// Class that maps generic arguments of the enclosed class/method the generic arguments of the closure.
```

Comment:
Is there a "to" missing?
Class that maps generic arguments of the enclosed class/method **to** the generic arguments of the closure.